translations/mod-fqm-manager/en.json

```diff
@@ -335,5 +335,10 @@
   "entityType.drv_organization_details.expected_activation_interval": "Organization expected activation interval",
   "entityType.drv_organization_details.vendor_currencies": "Organization vendor currencies",
   "entityType.drv_organization_details.tax_id": "Organization tax ID",
-  "entityType.drv_organization_details.tax_percentage": "Organization tax percentage"
+  "entityType.drv_organization_details.tax_percentage": "Organization tax percentage",
+  "entityType.drv_organization_details.agreements": "Organization agreements",
+  "entityType.drv_organization_details.agreements.name": "Organization vendor terms name",
```

Comment:
There are two translations for each nested property. If you check the logs of mod-fqm-manager while you're querying against it, you should see warnings for missing translations.
```suggestion
  "entityType.drv_organization_details.agreements.name": "Vendor terms name",
  "entityType.drv_organization_details.agreements.name._qualified": "Organization vendor terms name",
```

Here's how they differ:
- The normal `.name` is used as the inner column label, and likely needs the parent label to make sense (e.g. "Notes" doesn't make sense alone unless it's inside "Vendor terms"):
![image](https://github.com/folio-org/mod-fqm-manager/assets/8005215/707fea44-956e-491a-ba68-7d90f0e255b6)
- The `._qualified` is shown in the query builder/etc and should make sense alone, e.g. "Vendor terms notes"